src/com/amazon/ion/benchmark/Main.java

```diff
@@ -41,7 +42,8 @@
             + "[--ion-use-symbol-tokens <bool>]... [--paths <file>] "
             + "[--ion-use-lob-chunks <bool>]... [--ion-use-big-decimals <bool>]..."
             + "[--json-use-big-decimals <bool>]... <input_file>\n"
-
+        
+        + "  ion-java-benchmark generate <size> <type> <file>\n" // generate command 
```

Comment:
As we discussed, named options will be easier to use. E.g. for `<size>` something like `--data-size <size>`.